docs/build/js-client/1-js-client.md

```diff
@@ -1,11 +1,5 @@
 # Introduction
 
-Fluence is a platform that focuses on peer-to-peer (P2P) networks, which allows for the creation of decentralized applications. One of the main features of Fluence is the use of Aqua code, which is a programming language that enables developers to write code for the Fluence network.
+In order to interact with peers of a peer-to-peer network, e.g., to execute an Aqua script, the "client" itself has to be a peer. Such client peers don't have to be publicly accessible or long-running in nature, but they do need to follow the protocol. In case of the Fluence peer-to-peer networks, browsers, command line or node apps are available client-peer built on [js client](https://github.com/fluencelabs/js-client).
```

Comment:
For me, the second sentence sounds slightly ambiguous. What about the following one? "In case of the Fluence peer-to-peer networks, browsers, command line or node apps are built on the [JS client](https://github.com/fluencelabs/js-client)."